includes/wikia/GlobalFunctions.php

```diff
@@ -1658,3 +1658,15 @@ function wfForceBaseDomain( $url, $targetServer ) {
 	$finalUrl = http_build_url( $url, [ 'host' => $finalHost, ] );
 	return WikiFactory::getLocalEnvURL( $finalUrl );
 }
+
+/**
+ * @return bool
+ */
+function wfIsCurrentWikiAFandomComWiki() {
```

Comment:
Well, IIRC I wanted to name `wfHttpsEnabledForURL` something like this one but you stopped me and wanted it to be more generic :stuck_out_tongue: If the logic there changes and more URLs are https enabled then those hreflang links will break. IMO we need to choose either deduplication or being generic.

Reply:
I more meant reuse code, but make it named for a purpose so it's more resistant to implementation changes. So, I still think `wfHttpsEnabledForURL` is better than `wfIsCurrentWikiAFandomComWiki`, since presumably, this could also involve other domains, like .org in the future, at which point, `wfHttpsEnabledForURL` still wouldn't need to have the method name changed, but `wfIsCurrentWikiAFandomComWiki` would. Here, I'm thinking whether we want to say this is only enabled on fandom.com, or if it's only enabled on a wiki that is migrated/on HTTPS by default, etc. in which case you can use a generic name, without duplicating code. `wfIsCurrentWikiAFandomComWiki` is the kind of name I want to avoid for a method ideally. :) Even if this is a separate method, I'd make one of them reuse the other so one is just an alias to the other.

Reply:
Makes sense, how about https://github.com/Wikia/app/pull/16406/commits/e63e92d3969f1f40ad4bcf3137d42a1255790847?